publication-model-testing/src/main/java/no/unit/nva/model/testing/PublicationContextBuilder.java

```diff
@@ -51,6 +51,7 @@
 import nva.commons.core.JacocoGenerated;
 
 @JacocoGenerated
+@SuppressWarnings({"PMD.CouplingBetweenObjects", "PMD.NcssCount"})
```

Comment:
We need a task for the NcssCount too as this is often a real comprehension issue